scalafmt-tests/shared/src/test/scala/org/scalafmt/config/ScalafmtConfigTest.scala

```diff
@@ -20,16 +22,17 @@ class ScalafmtConfigTest extends FunSuite {
 
   test("file overrides") {
     val config = ScalafmtConfig.fromHoconString(
-      """
-        |newlines.source = fold
-        |newlines.topLevelBodyIfMinStatements = [before,after]
-        |fileOverride {
-        |  "glob:**/src/test/scala/**" {
-        |    newlines.source = unfold
-        |    newlines.topLevelBodyIfMinStatements = []
-        |  }
-        |}
-        |      """.stripMargin,
+      s"""
+         |newlines.source = fold
+         |newlines.topLevelBodyIfMinStatements = [before,after]
+         |fileOverride {
+         |  "glob:${PlatformCompat
+          .fixPathOnNativeWindows("**/src/test/scala/**")}" {
```

Comment:
can users still specify paths using forward slash? the config file is shared between platforms, and they can't have one for windows and another for unix.

Reply:
Turns out the problem I was trying to fix here was found elsewhere. I had to remove 2 uses of .asFilename compatibility tool (one used on fileOverride, one for include/excludePaths), as it messed with scala native's glob implementation. That asFilename method replaced unix file separator with Windows one, but it seems to work even on JVM just as well without it